### ISSUE TITLE
[BUG] Error in StateMet diagnostic variable name

### ISSUE THREAD

**Issue by cdholmes** [geoschem/geos-chem#1487]
### Description of the problem
Two of the StateMet diagnostic variables have incorrect or misleading names in the HISTORY netCDF output. This has no impact on any calculations within the model, but model users could be confused with the current names. In NetCDF files containing these variables, AIRVOL is described as "Volume of dry air in grid box" and BXHEIGHT is described as "Grid box height (w/r/t dry air)." In spite of these names, neither of these variables uses any dry air quantities and calculations with both dry and moist quantities should use the same volume and height. 

### Solution
I recommend using the following descriptions in Headers/state_met_mod.F90 
```F90 
      CASE ( 'AIRVOL' )
          IF ( isDesc  ) Desc  = 'Volume of grid box'

      CASE ( 'BXHEIGHT' )
          IF ( isDesc  ) Desc  = 'Grid box height'
```
### GEOS-Chem version
13.4 (at least)

**Comment by yantosca** [comment]
Thanks @cdholmes.  We can get this into 14.0.2, as it's a simple fix.

**Comment by yantosca** [comment]
@lizziel, do you have any comments about this update?

Also tagging @Jourdan-He and @SaptSinha

**Comment by yantosca** [comment]
Integration tests pending

**Comment by yantosca** [comment]
We can now close this issue as PR #1490 has been merged into the 14.0.2 development stream.

**Comment by lizziel** [comment]
Sorry to be so late to getting back on this. The updates look fine to me. I'm not sure why that language snuck in there in the first place since it is indeed confusing. If interested, users can see how `BXHEIGHT` is computed in `calc_met_mod.F90` where there is a detailed description of the formula (hypsometric equation using virtual temperature and dry air molecular weight).
